CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 
 - Wide characters sometimes being cut off
+- Preserve vi mode across terminal `reset`
```

Comment:
Since we use it with capital V in config, we should use capital here as well?

```suggestion
- Preserve Vi mode across terminal `reset`
```

Reply:
In the past we don't usually have it capitalized? I think officially vi isn't capitalized either?